master/internal/webhooks/postgres_webhook.go

```diff
@@ -272,6 +280,121 @@ func (l *WebhookManager) deleteWebhook(ctx context.Context, id WebhookID) error
 	return nil
 }
 
+func handleCustomTriggerData(ctx context.Context, data CustomTriggerData, experimentID int, trialID *int) error {
+	var m struct {
+		bun.BaseModel `bun:"table:experiments"`
+		model.Experiment
+		ConfigBytes []byte `bun:"config"`
+	}
+	err := db.Bun().NewSelect().Model(&m).ExcludeColumn("username").Where("id = ?", experimentID).Scan(ctx)
```

Comment:
Do we need the `ExcludeColumn("username")` when that'd be omitted anyway since we pass in the model `m` in the select query?

Reply:
We do need to exclude "username", otherwise there will be runtime error because we defined username in `model.Experiment` but not in DB

Reply:
Ahh gotcha! Makes sense that if it's an exported field in the embedded type